data/xml/2008.amta.xml

```diff
@@ -0,0 +1,543 @@
+      <author><first>Carol</first><last>van Ess-Dykema</last></author>
+      <author><first>Helen G.</first><last>Gigley</last></author>
+      <author><first>Stephen</first><last>Lewis</last></author>
+      <author><first>Emily</first><last>Vancho Bannister</last></author>
+      <url hash="413acf65">2008.amta-govandcom.24</url>
+      <pages>457-463</pages>
+      <abstract>This paper describes the strategic vision for a new translation management workflow for the US Government’s National Virtual Translation Center (NVTC). The paper also describes past, current, and planned experiments validating the vision, along with experiment results to-date. The most salient features of the new workflow include the embedding of translation technology at the front end of the workflow (e.g., translation memory technology, specialized lexicons, and machine translation), technology-generated “seed translation”, a new human work role called “paralinguist” to assess the “seed translation” and assign an appropriate translator/post-editor, and new human translation strategies including federated search of online dictionaries and collaborative translation.</abstract>
+    </paper>
+    <paper id="25">
+      <title>Applicability of Resource-based Machine Translation to Airplane Manuals</title>
+      <author><first>Eiko</first><last>Yamamoto</last></author>
+      <author><first>Akira</first><last>Terada</last></author>
+      <author><first>Hitoshi</first><last>Isahara</last></author>
+      <url hash="608b3a71">2008.amta-govandcom.25</url>
+      <pages>464-469</pages>
+      <abstract>Machine translation (MT) has been studied and developed since the advent of computers, and yet is rarely used in actual business. For business use, rule-based MT has been developed, but it requires rules and a domain-specific dictionary that have been created manually. On the other hand, as huge amounts of text data have become available, corpus-based MT has been actively studied, particularly corpus-based statistical machine translation (SMT). In this study, we tested and verified the usefulness of SMT for aviation manuals. Manuals tend to be similar and repetitive, so SMT is powerful even with a small amount of training data. Although our experiments with SMT are at the preliminary stage, the BLEU score is high. SMT appears to be a powerful and promising technique in this domain.</abstract>
+    </paper>
+    <paper id="26">
+      <title>Applications of <fixed-case>MT</fixed-case> during Olympic Games 2008</title>
+      <author><first>Chengqing</first><last>Zong</last></author>
+      <author><first>Heyan</first><last>Huang</last></author>
+      <author><first>Shuming</first><last>Shi</last></author>
+      <url hash="17ccaae6">2008.amta-govandcom.26</url>
+      <pages>470-479</pages>
+    </paper>
+  </volume>
+  <volume id="additional" ingest-date="2021-05-19">
+    <meta>
+      <booktitle>Additional Papers not in Proceedings</booktitle>
```

Comment:
Let's call this "Proceedings of the Eighth Conference of the Association for Machine Translation in the Americas: Additional Papers"